src/CSnakes.Runtime/Python/PyObject.cs

```diff
@@ -157,6 +158,25 @@ public string GetRepr()
         }
     }
 
+    /// <summary>
+    /// Is the Python object None?
+    /// </summary>
+    /// <returns>true if None, else false</returns>
+    public virtual bool IsNone()
+    {
+        return CPythonAPI.IsNone(this);
+    }
+
+    public static PyObject None
+    {
+        get
+        {
+            if (none is null)
+                throw new InvalidOperationException("Python is not initialized. You cannot call this method outside of a Python Environment context.");
+            return none;
+        }
+    }

```

Comment:
```suggestion
    public static PyObject None => none ??
                throw new InvalidOperationException("Python is not initialized. You cannot call this method outside of a Python Environment context.");

```

I **think** that is right.